app/webpack.production.js

```diff
@@ -4,31 +4,13 @@ const common = require('./webpack.common')
 
 const webpack = require('webpack')
 const merge = require('webpack-merge')
-const ExtractTextPlugin = require('extract-text-webpack-plugin')
-const BabelPlugin = require('babel-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
 
 const config = {
+  mode: 'production',
   devtool: 'source-map',
-  plugins: [
-    new BabelPlugin({
-      test: /\.js$/,
-      sourceMaps: true,
-      compact: true,
-      minified: true,
-      comments: false,
-      presets: [
-        [
-          'minify',
-          {
-            evaluate: false,
-          },
-        ],
-      ],
-    }),
-    new webpack.optimize.ModuleConcatenationPlugin(),
```

Comment:
Was removal of the `ModuleConcatenationPlugin` intentional and if so why? Are the measurements from https://github.com/desktop/desktop/pull/2512 no longer applicable?

Reply:
Yeah, because the docs for [`mode: 'production'`](https://webpack.js.org/concepts/mode/#mode-production) make it look like it was already covered:

```diff
// webpack.production.config.js
module.exports = {
+  mode: 'production',
-  plugins: [
-    new UglifyJsPlugin(/* ... */),
-    new webpack.DefinePlugin({ "process.env.NODE_ENV": JSON.stringify("production") }),
-    new webpack.optimize.ModuleConcatenationPlugin(),
-    new webpack.NoEmitOnErrorsPlugin()
-  ]
}
```